.pep8

```diff
@@ -0,0 +1,5 @@
+[flake8]
```

Comment:
why is this file added? there is already a config for `flake8` in `tox.ini`.  I'm also not so quick to ignore all those other things.